app/src/main/java/com/vuongnq14798/vuongmp3/ui/splash/SplashActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.vuongnq14798.vuongmp3.ui.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import com.vuongnq14798.vuongmp3.MainActivity
+import com.vuongnq14798.vuongmp3.R
+import com.vuongnq14798.vuongmp3.util.Constants
+
+class SplashActivity: AppCompatActivity() {
+
+    private val splashTime = Constants.SPLASH_TIME
+    private lateinit var mHandler: Handler
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+
+        mHandler = Handler()
+        mHandler.postDelayed({
+            startActivity(Intent(applicationContext, MainActivity::class.java))
+            finish()
+        }, splashTime)
+    }
```

Comment:
ông có thể tham khảo ở đây cách làm splash screen đúng cách: https://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154
để việc tạo intent thuộc về MainActivity theo conventions : https://github.com/framgia/coding-standards/blob/master/vn/android/codingstyleguide.md#211-%C4%91%E1%BB%91i-s%E1%BB%91-trong-activity-v%C3%A0-fragment